INN/umbrella-ipbs
# Homepage top styles

## Changes

This PR styles the homepage top:

- font scales between 28px and 40px 
- blue background is transparent at desktop sizes and overlaid on the featured image
- on mobile sizes, the blue area drops below the featured image and loses its transparency
- on mobile sizes, the Quick Links menu drops below the featured story's excerpt
- removes top term from template

![Screen Shot 2020-01-24 at 22 06 56](https://user-images.githubusercontent.com/1754187/73115730-15fdfd80-3ef8-11ea-9b9e-e1fed1591ad6.png)
![Screen Shot 2020-01-24 at 22 07 58](https://user-images.githubusercontent.com/1754187/73115731-16969400-3ef8-11ea-94dd-2cd53c166343.png)


## Why



Resolves #55 

## Testing/Questions

Features that this PR affects:

- Homepage


Questions that need to be answered before merging:

- [x] Do we need a fallback image if the homepage top story lacks a featured image?

Steps to test this PR:

1. Switch to the IPBS Homepage template in Theme Options > Layout
2. In Appearance > Menus, make sure the Homepage Quick Links menu is populated. If no such menu exists, go to Menu Locations and make one.
3. Make sure the most-recent post on the site has a featured image
4. View the homepage. ![resizing](https://gifs.benlk.com/resizing.gif)
